docs/DafnyRef/Types.md

```diff
@@ -232,7 +232,14 @@ There are also literals for some of the reals.  These are
 written as a decimal point with a nonempty sequence of decimal digits
 on both sides, optionally prefixed by a `-` character.
 For example, `1.0`, `1609.344`, `-12.5`, and `0.5772156649`.
-Real literals using exponents are not supported in Dafny (at present).
+Real literals using exponents are not supported in Dafny. For now, you'd have to write your own function for that, e.g. 
+```dafny
+// realExp(2.37, 100) computes 2.37e100
+function method realExp(r: real, e: int) {
+  if e == 0 then r
+  else if e < 0 then realExp(r/10, e+1)
+  else realExp(r*10, e)
```

Comment:
```suggestion
function method realExp(r: real, e: int): real decreases if e > 0 then e else -e {
  if e == 0 then r
  else if e < 0 then realExp(r/10.0, e+1)
  else realExp(r*10.0, e-1)
}
```
I tried my previous function in Dafny and it had several typo and one verification error (the function was not terminating on the final else branch)